microk8s-resources/wrappers/run-containerd-with-args

```diff
@@ -30,9 +30,16 @@ app=containerd
 
 
 RUNTIME="runc"
+RUNTIME_TYPE="io.containerd.runc.v1"
 SNAPSHOTTER=$(snapshotter)
 
+if mount | grep -q 'cgroup2 on /sys/fs/cgroup'; then
```

Comment:
Within LXD this seems to get mounted as `none on /sys/fs/cgroup` and fails this check.